spec/services/estate_visit_query_spec.rb

```diff
@@ -151,9 +170,14 @@
       FactoryBot.create(:visit, :requested, prison: prison)
     end
 
+    let!(:old_visit) do
+      FactoryBot.create(:visit, :requested, prison: prison, created_at: 7.months.ago, updated_at: 7.months.ago)
```

Comment:
no need to use `FactoryBot` just call `create(:visit)`

Reply:
basically if you see `FactoryBot` in front of a `create` or `build` or `build_stubbed` and so on rip it off!